frontend/src/Root.js

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
-import { BrowserRouter } from 'react-router-dom';
+import { Router } from 'react-router-dom';
 import { Provider } from 'react-redux';
 import { persistStore } from 'redux-persist';
+import browserHistory from './common/utils/routerUtils';
```

Comment:
./common/utils/routerUtils should be named ./common/browserHistory

Reply:
We might have to add other things on this. Are you suggesting we should rename it to browserHistory now, and if we add some other objects to this then change it to routerUtils?

Reply:
routerUtils in this case has created a global object "browserHistory", so the above naming would be more suitable
You can always use non-default exports for other objects that will be added.